consoles/remoteVnc.pm

```diff
@@ -38,6 +38,16 @@ sub activate {
         });
 }
 
+sub reset {
+    my ($self) = @_;
+
+    if ($self->{activated}) {
+        $self->SUPER::disable();
```

Comment:
but if this code is a thing now in console, the base class should call a deactivate function - which is then different in ipmi and vnc.

And it makes you wonder if this shouldn't be in vnc_base if it's important

Reply:
Should I move this to `vnc_base` then? I really haven't seen problems with the other consoles, but don't think this will do harm.